doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -322,12 +292,16 @@ Requests to the environment
             Search (?x * _ + ?x * _)%Z outside OmegaLemmas.
 
 
-.. cmd:: SearchHead @term
+.. cmd:: SearchHead @one_term {? {| inside | outside } {+ @qualid } }
+
+   Displays the name and type of all hypotheses of the
+   selected goals (if any) and theorems of the current context that have the
+   form :n:`{* P__n -> } (@one_term t__1 ... t__m)`.
+   It's useful for finding the names of library lemmas.
```

Comment:
Doesn't seem useful to me to repeat this:
```suggestion
```

Reply:
Sorry.  IIRC that sentence appeared in more than one place.  I may have removed the wrong one (and the others may not have been marked).

Reply:
I suggest keeping it only once, in the description of `Search`.